package.json

```diff
@@ -77,9 +77,8 @@
     "@babel/traverse": "^7.23.2"
   },
   "engines": {
-    "npm": ">=7.0.0",
-    "node": ">=16.0.0"
+    "node": ">=18.14.1"
```

Comment:
bumped minimum required node version as 18 is the current LTS, and 16 is no longer supported by many deps

make sure to run fresh `yarn install` after upgrading node versions